jina/flow/base.py

```diff
@@ -1031,7 +1031,7 @@ def start(self):
                 os.environ[k] = str(v)
 
         for k, v in self:
-            if not getattr(v.args, 'external', False):
+            if not getattr(v.args, 'external', False) and not v.is_sandbox:
```

Comment:
Can u perhaps add a `property` in `BasePod` 

```python
@property 
def external(self)
    return getattr(self.args, 'external', False) or self.is_sandbox
```

and use it in this check?

Reply:
addressed